README.md

```diff
@@ -1,5 +1,43 @@
-# Pelican ASF Website
+# Apache Gora Website
 
-See [ASF-Pelican](https://infra.apache.org/asf-pelican.html)
+Welcome to the Apache Gora website repository! This README provides guidance on building the website locally for development or testing purposes.
 
-See [migration notes](migration/README.md)
+## Building Locally
+
+To build this website locally, follow these steps:
+
+### Prerequisites
+- Ensure that you have [Python 3](https://python.org) installed on your system.
+
+### Setup
+1. Clone this repository to your local machine.
+
+```bash
+git clone https://github.com/apache/gora-site.git
+```
+
+2. (Optional) Set up a new Python virtual environment.
+
+```bash
+python3 -m venv env
+source env/bin/activate
+```
+
+3. Install the required Python packages using pip.
+
+```bash
+pip install -r requirements.txt
+```
+> Note: You may also need to run `pip install pelican[markdown]`
```

Comment:
```
(env) ➜  gora-site git:(main) pip install pelican[markdown]
zsh: no matches found: pelican[markdown]
```

Reply:
Depending on the system, will need to escape it in quotes: `pip install "pelican[markdown]"`, I can update the README if need be

Reply:
Resolved. Once I resolve the rest, I will squash.